### PR TITLE
cs-43660 live preview 1.0 and 2.0 in ts

### PR DESCRIPTION
**Pull request by harshithad0703**
- support for live preview 1.0 and 2.0
- added live preview query function to stack
- added unit and api test case for live preview

### CODE REVIEW

File: src/lib/stack.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export class Stack {
   constructor(client: AxiosInstance, config: StackConfig) {
     this._client = client;
     this.config = config;
+    this.config.live_preview = config.live_preview
```
Comment (nadeem-cs) [inline src/lib/stack.ts:18]: @harshithad0703 , can you remove the line as it seems redundant - the above live should take care of the live_preview property.

Comment (harshithad0703) [reply src/lib/stack.ts]: Removed that line
Thank you

**Review by nadeem-cs — APPROVED** [review]
Approved.

**Review by aravindbuilt — APPROVED** [review]

**Review by sunil-lakshman — APPROVED** [review]
Approved

**Pull request merged**